[Release notes] Cloud changelogs catch up (#960)
seqeralabs/docs: changelog/seqera-cloud/v25.3.0_cycle32.md

---
title: Seqera Cloud v25.3.0_cycle32
date: 2025-11-18
tags: [seqera cloud]
---

## Bug fixes

- Fixed an issue where launching shared pipelines from another workspace would fail to load pipeline configuration data, resulting in an empty launch form. The launch form now correctly retrieves and displays all pipeline configuration when launching shared pipelines across workspaces.

> - Fixed an issue where OIDC authentication errors containing spaces or special characters caused application crashes instead of displaying proper error messages. Authentication errors during login now display correctly to users, preventing application failures.

Reviewer: ```suggestion
- Fixed OIDC authentication errors with spaces or special characters causing application crashes instead of displaying proper error messages. Authentication errors during login now display correctly to users, preventing application failures.
```

> - Fixed an issue where OIDC authentication errors containing spaces or special characters caused application crashes instead of displaying proper error messages. Authentication errors during login now display correctly to users, preventing application failures.

Reviewer 2: ```suggestion
- Fixed OIDC authentication errors with spaces or special characters crashing the application instead of displaying error messages. Authentication errors during login now display correctly to users, preventing application failures.
```
